src/modules/Collection.js

```diff
@@ -0,0 +1,48 @@
+export default class Collection {
+  constructor(localStorageItem, items) {
+    this.localStorageItem = localStorageItem;
+    this.items = items || [];
+  }
+
+  addItem = (description) => {
+    const completed = false;
+    const index = this.items.length + 1;
+    this.items.push({
+      description,
+      completed,
+      index,
+    });
+    this.items.sort((a, b) => a.index - b.index);
+    localStorage.setItem(this.localStorageItem, JSON.stringify(this.items));
```

Comment:
These two lines are repeated in lines 24-25 and lines 45-46. You can kindly refactor in a function to DRY.

Reply:
Thanks! Now it's corrected using a function.